webdriver_manager/core/http.py

```diff
@@ -29,6 +31,18 @@ def __init__(self):
         self._ssl_verify = ssl_verify()
 
     def get(self, url, **kwargs) -> Response:
-        resp = requests.get(url=url, verify=self._ssl_verify, **kwargs)
+        resp = requests.get(url=url, verify=self._ssl_verify, stream=True, **kwargs)
         self.validate_response(resp)
+
+        total = int(resp.headers.get("Content-Length", 0))
```

Comment:
can we move this to a method like show_progress or something like that?

Reply:
Sure thing, I added it in utils.py though since it was static anyway